api/envoy/admin/v3/certs.proto

```diff
@@ -58,7 +58,7 @@ message CertificateDetails {
   repeated SubjectAlternateName subject_alt_names = 3;
 
   // Minimum of days until expiration of certificate and it's chain.
-  uint64 days_until_expiration = 4;
+  int64 days_until_expiration = 4;
```

Comment:
I'm honestly not sure if we can change this or not. I think it's probably ok because this is just used to render json on  the admin endpoint, but it's convievable that someone is parsing it using this proto. 

If we changed it at all it should be changed to a WKT, not a signed integer. 

For this output can't we just set to 0 if expired to avoid a change?

cc @envoyproxy/api-shepherds

Reply:
0 is ok to me, while is not more obvious than -1 I think,

Reply:
+1 on setting it to 0, as long as it has the same semantics (i.e., if an admin reads this value, a value -of 0 or -1 does not impact the understanding here).

If the semantics are different, then I'd say that what matters is whether the cert has expired or not, and maybe some other field needs to be added to state that.

Reply:
Updated, please review.